### PR TITLE
picasso update

### PR DESCRIPTION
**Pull request by Martin-Kiss**
- pica tokenomics
- token transparency
- uses cases
- legal disclaimers

### CODE REVIEW

File: docs/docs/parachains/picasso/governance.md [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+the Sudo council will be burned and the First Council will then be in charge of governance. 
+The First Council is an on-chain entity made up of 11 senior team members from and our supporters. 
+The Council members also control Picasso’s multi-sig wallet.
+
+When the Picasso parachain is live, each council member will be represented as an on-chain account on Polkadot.js. 
+Members of the First Council will consist of:
+- 0xBrainJar, Composable Founder/CEO
+- Jeff Smith, Composable General Counsel (GC)
+- Eoin Whelan, Composable CFO
+- Seun Lanlege, Composable Rust Substrate Lead
```
Comment (0xBrainjar2) [inline docs/docs/parachains/picasso/governance.md]: bridging protocol lead

**Review by 0xBrainjar2 — APPROVED** [review]

File: docs/docs/parachains/picasso/governance.md (hunk 1 above)
Comment (Martin-Kiss) [reply docs/docs/parachains/picasso/governance.md]: According to the content update doc the polkassembly guide isn’t ready(?) It’s marked as pending review.

**Review by KaiserKarel — APPROVED** [review]

File: docs/docs/parachains/picasso/token-transparency.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+
+Treasury Multisig: 5wCLknMQkBGmuTQxsxYAyevpa37mMCkFLaqaYNcowsSnR7CJ 
+
+Team Tokens Multisig: 5wyQr1huDp4mKAKA95ihXRqWu8hLkqjiNib8GaKFzuEMKEk3
+
+Ecosystem Incentives Multisig : 5ytUzQtNj7vibyr79cqUENuDLNUnTsgDAJMDefhBywRTNrDU
+
+Liquidity Programs Multisig : 5ytrBrTBma4kKfmEXGoKGMVjfyTnKgpvXaiwwCbXCbdZQK25
+
+Treasury Pallet for accruing fees: 5wCLknMQkBGmuTQxsxYAyevpa37mMCkFLaqaYNcowsSnR7CJ
```
Comment (0xBrainjar2) [inline docs/docs/parachains/picasso/token-transparency.md:79]: This is incorrect, it is the treasury pallet, treasury pallet is different from the treasury multisig, plz fix

File: docs/docs/parachains/picasso/token-transparency.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+### Protocol Controlled Addresses
+
+#### Crowdloan Pallet Funds
+
+Are located at the crowdloan pallet:
```
Comment (0xBrainjar2) [inline docs/docs/parachains/picasso/token-transparency.md:85]: when we have this address, please provide 

and also link to the csv of the rewards as well

File: docs/docs/parachains/picasso/tokenomics.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+|                        Treasury                         |  1,800,297,200 |       ~18%       | Approximately 18% of PICA’s supply is allocated to account for the long-term sustainability of the Picasso Network. Picasso Network treasury receives transaction fees and is governed by the Picasso General Council, which is intended to later be governed by PICA token holders. 75% of spent network fees will automatically flow into this fund (the remaining 25% of spent fees are intended to be distributed to collators).** |
+|               Future Employee Incentives                |    200,000,000 |        2%        | 2% of PICA is intended to be reserved for future employee incentives. Future issues from this pool will be subject to a 4-year vesting schedule from network launch or grant date (whichever is later) with a 1-year cliff and monthly vesting thereafter.**                                                                                                                                                                           |
```
Comment (0xBrainjar2) [inline docs/docs/parachains/picasso/tokenomics.md:14]: the 2 and 16 are supposed to be subcategories of the 18%

**Review by 0xBrainjar2 — CHANGES_REQUESTED** [review]

File: docs/docs/parachains/picasso/token-transparency.md (hunk 2 above)
Comment (Martin-Kiss) [reply docs/docs/parachains/picasso/token-transparency.md:79]: @KaiserKarel could you provide the above?

File: docs/docs/parachains/picasso/tokenomics.md (hunk 4 above)
Comment (Martin-Kiss) [reply docs/docs/parachains/picasso/tokenomics.md:14]: Best I can for this in markdown short term is put something like "Future employee incentives" (as part of Treasury)
Additional Allocations (as part of Treasury)

File: docs/docs/parachains/picasso/token-transparency.md (hunk 3 above)
Comment (Martin-Kiss) [reply docs/docs/parachains/picasso/token-transparency.md:85]: @0xBrainjar2 Where can I get this or who can I ask for this address?

**Review by KaiserKarel — APPROVED** [review]

**Pull request merged**